mirar/pipelines/winter/load_winter_image.py

```diff
@@ -259,8 +270,19 @@ def load_raw_winter_mef(
     """
     primary_header, split_data, split_headers = open_mef_fits(path)
 
-    primary_header = clean_header(primary_header)
-
+    try:
+        primary_header = clean_header(primary_header)
+    except KeyError:
+        logger.warning(f"Could not clean header for {path}, missing keywords")
+        primary_header[OBSCLASS_KEY] = "test"
+        primary_header["COADDS"] = 1
+        primary_header["CALSTEPS"] = ""
+        primary_header["PROCFAIL"] = 1
+        primary_header[TARGET_KEY] = "test"
```

Comment:
should this stuff stay in here? Especially the date?

Reply:
Yes, iirc if all these fields are not in there and it fails at this step, it crashes the pipeline without processing other images. The date is a dummy before start of operations.